src/pages/coaches/francisco-gomes.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Francisco Gomes
+title: Software Engineer
+companyName: Sainsbury's
+companyLink: https://www.sainsburys.co.uk/
+gitHub: https://github.com/franciscomcg
+twitter: https://twitter.com/francisgomes83
+linkedIn: https://uk.linkedin.com/in/francisco-gomes-webdev
+blockquote: While teaching, the technology itself is not life-changing. It's the passion and commitment.
+imageDescription: React GraphQL Academy coach Francisco Gomes, smiling in vineyard
+imageSrc: https://firebasestorage.googleapis.com/v0/b/reactjsacademy-react.appspot.com/o/coach_profile_images%2FFrancisco-photo%20(1).jpg?alt=media
+tags: []
+---
+
+ <ul>
```

Comment:
@ric9176 why don't we use markdown for the list? https://www.markdownguide.org/basic-syntax/#unordered-lists